skiko/src/commonMain/kotlin/org/jetbrains/skia/ColorSpace.kt

```diff
@@ -12,10 +12,35 @@ class ColorSpace : Managed {
 
         val sRGB = ColorSpace(_nMakeSRGB(), false)
         val sRGBLinear = ColorSpace(_nMakeSRGBLinear(), false)
-        val displayP3 = ColorSpace(_nMakeDisplayP3(), false)
+        val displayP3: ColorSpace = makeRGB(TransferFunction.sRGB, Matrix33.displayP3ToXYZD50)!!
```

Comment:
Let's avoid `!!`. At least you can use something like `requireNotNull` or `?: error(...)`

Reply:
Agreed, changed it to `requireNotNull`.